omnibus/package-scripts/agent/postrm

```diff
@@ -29,6 +29,9 @@ if [ -f "/etc/debian_version" ] || [ "$DISTRIBUTION" = "Debian" ] || [ "$DISTRIB
             rm -rf $LOG_DIR
             rm -rf $CONFIG_DIR
         ;;
+        remove)
+            rm -f $CONFIG_DIR/install_info
```

Comment:
Let's add `|| true` at the end, so we don't fail if the file has already been removed.

Note that any command that fails, stops the script since we are using `set -e`

Reply:
I think the `-f` prevents it from failing (the manual says `ignore nonexistent files and arguments`) but I can add it just in case if you think it is better.